### PR TITLE
chore: Add sensible dotfiles

### LINKED ISSUE #11: Add sensible dotfiles
- [x] `.editorconfig`
- [x] `tslint.json`
- [x] `.gitignore`
- [x] `.nvmrc`
- [x] `.npmrc`

### PR DESCRIPTION
**Pull request by simonplend**
Fixes #11.

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+package-lock=false
```
Comment (sjparkinson) [inline .npmrc]: What does this do?

Comment (simonplend) [reply .npmrc]: https://github.com/financial-times-sandbox/tako/pull/15/commits/47c58298241d130fc43147d6fb88d79574a46b4f

Comment (simonplend) [reply .npmrc]: Have added a comment to `.npmrc` to reference the npm config docs.

**Comment by simonplend** [comment]
I've added `tslint` and the config for it lives in `tslint.json`. I've set it up to use the [`tslint:latest`](https://github.com/palantir/tslint/blob/master/src/configs/latest.ts) rules preset and the [`tslint-config-prettier`](https://github.com/alexjoverm/tslint-config-prettier#readme) rules preset, which removes any rules that might conflict between `tslint` and `prettier`, ensuring a neat separation of concerns.

We should...

- [ ] Find out if anyone else in Next-land has a config that we like the look of and nab that

...or...

- [ ] Decide if we like the `tslint:latest` rules
- [ ] Take a look at the [`.eslintrc.js` in `n-gage`](https://github.com/Financial-Times/n-gage/blob/master/dotfiles/.eslintrc.js) and see if there are equivalent rules in `tslint` that we should add to the config - enforcing single quotes seems like a good candidate, for example
- [ ] See what other [`tslint`](https://palantir.github.io/tslint/rules/) rules we want to add

I suggest any `tslint.json` config changes happen outside of this PR so we can get this one merged.

**Comment by sjparkinson** [comment]
This all looks 👍.

There are a few TypeScript projects around the place, consent is one I believe. Worth a look.

**Comment by simonplend** [comment]
Existing next project `tslint.json` config files:

- [next-consent-proxy/tslint.json](https://github.com/Financial-Times/next-consent-proxy/blob/master/tslint.json)
- [n-user-api-client/tslint.json](https://github.com/Financial-Times/n-user-api-client/blob/master/tslint.json)
- [next-fow-api//tslint.json](https://github.com/Financial-Times/next-fow-api/blob/master/tslint.json)
- [n-fastly-purger/tslint.json](https://github.com/Financial-Times/n-fastly-purger/blob/master/tslint.json)

(Spoiler: they're all the same!)

**Comment by simonplend** [comment]
A final few to-dos before I merge this PR:

- [ ] Enforce single-quotes, ala [Next convention](https://github.com/Financial-Times/n-gage/blob/master/dotfiles/.eslintrc.js#L31) with `tslint` and `prettier`
- [ ] Add notes to the `README` about the tool we're using

**Pull request merged**